mcs/class/corlib/System/Console.iOS.cs

```diff
@@ -7,7 +7,7 @@
 // Copyright 2012-2014 Xamarin Inc. All rights reserved.
 //
 
-#if FULL_AOT_RUNTIME
```

Comment:
I think this should be `#if FULL_AOT_RUNTIME || MONOTOUCH`, since there may be full AOT runtimes which aren't MonoTouch.